tecken/upload/utils.py

```diff
@@ -23,8 +22,6 @@
 
 logger = logging.getLogger("tecken")
 
-# FIXME(willkg): This downloader needs to point to all the things that can be downloaded
-# from so it can correctly invalidate the cache
```

Comment:
@willkg : I'm not sure I grok what this FIXME is about, but I gather it's related to caching that we're no longer doing. Is it appropriate to remove?

Reply:
You removed the use of this `downloader`, so the `downloader` line (line 28 I think) can be removed as well as this comment.

Reply:
For some context about the comment, it's marked as a FIXME because it hardcodes the symbol sources here. We really want the list of symbol sources to be configurable and not hardcoded in a bunch of places. We need the downloader here to cover all the possible sources (regular and try) so that it invalidates the cache correctly when someone uploads a file which might be in the cache as a 404. This fixed bug-1758566.